### ISSUE TITLE
Error on BGT 1.20 (20 Feb 19)

### ISSUE THREAD

**Issue by ALIENQuake** [Black-Wyrm-Lair/Drizzt-Saga#14]
In order to not forget about it: I'm facing an error during BGT installation

```
// Log of Currently Installed WeiDU Mods
// The top of the file is the 'oldest' mod
// ~TP2_File~ #language_number #component_number // [Subcomponent Name -> ] Component Name [ : Version]
~BGT/BGT.TP2~ #7 #0 // Baldur's Gate Trilogy - Rdzen: 1.20 (20 Feb 19)
~DRIZZTSAGA/DRIZZTSAGA.TP2~ #3 #0 // Saga Drizzta dla BG:EE/TuTu/BGT -> Obszary polaczone poprzez krawedzie map (domyslnie): v3.00
```

```
Copying and patching 1 file ...

Kompiluje skrypty i dialogi...
Compiling 39 scripts ...

[tb#_compile_eval_buffer/DRIZZTSAGA/Scripts/BAF/f_genefn.baf] PARSE ERROR at line 4 column 1-38
Near Text: )
        [StateCheck] argument [STATE_REALLY_DEAD] not found in [State.IDS]

[tb#_compile_eval_buffer/DRIZZTSAGA/Scripts/BAF/f_genefn.baf] PARSE ERROR at line 16 column 1-38
Near Text: )
        [StateCheck] argument [STATE_REALLY_DEAD] not found in [State.IDS]

[tb#_compile_eval_buffer/DRIZZTSAGA/Scripts/BAF/f_genefn.baf] PARSE ERROR at line 96 column 1-38
Near Text: )
        [StateCheck] argument [STATE_REALLY_DEAD] not found in [State.IDS]

[tb#_compile_eval_buffer/DRIZZTSAGA/Scripts/BAF/f_genefn.baf] PARSE ERROR at line 117 column 1-38
Near Text: )
        [StateCheck] argument [STATE_REALLY_DEAD] not found in [State.IDS]

[tb#_compile_eval_buffer/DRIZZTSAGA/Scripts/BAF/f_genefn.baf] PARSE ERROR at line 133 column 1-38
Near Text: )
        [StateCheck] argument [STATE_REALLY_DEAD] not found in [State.IDS]

[tb#_compile_eval_buffer/DRIZZTSAGA/Scripts/BAF/f_genefn.baf] PARSE ERROR at line 154 column 1-38
Near Text: )
        [StateCheck] argument [STATE_REALLY_DEAD] not found in [State.IDS]
Compiling 4 scripts ...
Copying and patching 1 file ..
```

**Comment by Roberciiik** [comment]
Hi @TotoR115, it seems that the file you have added in your last pull requests miss some states in IDS. What is the source game, that you copied the file from?

**Comment by TotoR115** [comment]
Hi @Roberciiik, 

I always install BG2fixpack, witch includes this state and many corrections to scripts, before reusing them... without it there could always remains some issues.

When I have copied the script from base bg2, it was already corrected and STATE_DEAD was replaced by STATE_REALLY_DEAD.

I am currently far from a PC so I can't do any PR before a while. But, and as this State exists for EE and works for bg2, the best solution is to add it with:
APPEND ~state.ids~ ~0x00000FC0 STATE_REALLY_DEAD~ UNLESS ~^0x00000FC0[ %TAB]+STATE_REALLY_DEAD[ %TAB%%LNL%%MNL%%WNL%]~

**Comment by ALIENQuake** [comment]
In that case, how about we just require BG2Fixpack?

**Comment by Roberciiik** [comment]
If there is a simple workaround of adding missing state, let's add it to the code and write suggestion about fixpack in the readme. Most oBG players will install fixpack either way. For internal testing purposes, it is easier to skip fixpack.

**Comment by ALIENQuake** [comment]
Yep, BGT is a much better place to require Fixpack.

@TotoR115 I've tested the provided code and while the symbol is properly added into override/stat.ide, I'm getting the same error.

**Comment by Roberciiik** [comment]
I've tested the provided fix and it works on BGT-v1.20.

**Comment by ALIENQuake** [comment]
Thanks!
